src/LowLevelMotionController.cpp

```diff
@@ -90,12 +90,13 @@ int main(int argc, char **argv)
 
     ROS_INFO("Low_Level_Motion_Control begin");
 
-    ros::NodeHandle nh("low_level_motion_controller");
+    ros::NodeHandle nh;
+    ros::NodeHandle param_nh ("low_level_motion_controller");
```

Comment:
I agree, but it's not really a matter of convention.  Doing it the other way requires that you use global names for topics which aren't in the node's namespace, which prevents you from running the node inside another namespace